### ISSUE TITLE
merge GO:0015039, GO:0004324 & GO:0016730? parent GO:0016651?

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#15359]
I hope I'm not coming across as a complete numpty now but should the terms

GO:0015039 NADPH-adrenodoxin reductase activity & 
GO:0004324 ferredoxin-NADP+ reductase activity 
GO:0016730 oxidoreductase activity, acting on iron-sulfur proteins as donors

be merged? Looks like the same thing, but "different" substrates? Even the substrates look similar e.g. human adrenodoxin FDX1 has synonym "Adrenal ferredoxin" and human ferredoxin FDX2 has synonym "Adrenodoxin-like protein". Both are modified with iron-sulfur clusters

 http://www.uniprot.org/uniprot/P10109 
http://www.uniprot.org/uniprot/Q6P4F2

**Comment by pgaudet** [comment]
Hi @Antonialock 

(1) 
GO:0015039 NADPH-adrenodoxin reductase activity &
GO:0004324 ferredoxin-NADP+ reductase activity
have different EC numbers; what's our rule - do we create different GO terms for each EC ? @hdrabkin ? 

(2) I suppose GO:0016730 oxidoreductase activity, acting on iron-sulfur proteins as donors
should have the partial EC 1.18.-.-

(3) On a related note, the parents of GO:0016699 oxidoreductase activity, acting on hydrogen as donor, iron-sulfur protein as acceptor seem wrong:: 
'oxidoreductase activity, acting on iron-sulfur proteins as donors'
AND 
'oxidoreductase activity, acting on hydrogen as donor': 

'oxidoreductase activity, acting on iron-sulfur proteins as donors' is not right, that's the acceptor. 


Pascale

**Comment by ukemi** [comment]
In general we do create separate terms for separate EC numbers. We also follow the EC classification.

**Comment by vanaukenk** [comment]
Summary of actions:

(1) Leave GO:0015039 and GO:0004324 as is since they have different EC numbers.

(2) Add EC 1.18. to GO:0016730

(3) Remove GO:0016730 'oxidoreductase activity, acting on iron-sulfur proteins as donors' is_a parent from GO:0016699 'oxidoreductase activity, acting on hydrogen as donor, iron-sulfur protein as acceptor' leaving GO:0016695 'oxidoreductase activity, acting on hydrogen as donor' as is_a parent

@pgaudet @hdrabkin - does this look alright to you?

**Comment by hdrabkin** [comment]
Yes, do not merge them as they have different substrate requrirements.
Only add 3 digit ec to grouping terms

**Comment by hdrabkin** [comment]
In general yes, one go term per ec; I can't think of any that have two 4 digit ecs for one term. I may be wrong.

**Comment by Antonialock** [comment]
ok, cheers, I thought the substrate could just be specified with an extension.

**Comment by vanaukenk** [comment]
Okay, I've done (3) above.
@hdrabkin - just so we all understand, why do we only add ec numbers with three or more digits to grouping terms?
Thx.

**Comment by hdrabkin** [comment]
Because they are too too broad. 
example 1.1.1 could map to all dehydrogenases. This wrecked havoc on the ec2go mapping that is now generated from the ontology file (@cmungall can explain better than I) ; 
and the grouping tems get the three digit ids, NOT 3+, which are specific activities.

**Comment by vanaukenk** [comment]
Okay, thanks @hdrabkin 
Might be good to just review this again on an editors call, and we should also write up an editor's doc on SOPs for adding xrefs if one doesn't already exist somewhere.....

**Comment by hdrabkin** [comment]
This is why I have a big ticket on removing 3 digits for specific activities; see #12988 ; in fact, I'm doing a bunch today when I see traffic decrease.

**Comment by vanaukenk** [comment]
I am closing this ticket for now.  I think anything that could be done has been done.
@ukemi @hdrabkin 
I did find some documentation for Enzyme xrefs here:
http://wiki.geneontology.org/index.php/Curator_Guide:_Enzymes_and_Reactions
I don't know how up-to-date this is, but it would be helpful to take a look and see if it should be included in the overall ontology editor's documentation.
Thx.
